libraries/GCS_MAVLink/GCS_Common.cpp

```diff
@@ -1702,14 +1702,6 @@ void GCS_MAVLINK::log_mavlink_stats()
         return;
     }
 
-    enum class Flags {
-        USING_SIGNING = (1<<0),
```

Comment:
need to do something sensible for these shifted forms?

Reply:
Done.

Sample output - search for "Auxiliary" in this file:

[LogMessages.rst.zip](https://github.com/ArduPilot/ardupilot/files/10486367/LogMessages.rst.zip)